torch/_tensor_docs.py

```diff
@@ -939,6 +939,19 @@ def add_docstr_all(method, docstr):
 See :func:`torch.conj`
 """)
 
+add_docstr_all('copysign',
+               r"""
+copysign(other) -> Tensor
+
+Returns a new tensor with the magnitude of :attr:`input` and the sign of :attr:`other` elementwise.
```

Comment:
Typically for method variants we just refer the reader to the function so we don't have duplicative documentation. So this sentence and the next one can be removed.

Reply:
It's probably nicer for the reader to not have to jump through a link if they're sort of familiar with what the function does but wants a quick reminder

Reply:
We should be consistent. If we add the description here we should do it for every other method variant, too.